app/controllers/content_assets_controller.rb

```diff
@@ -0,0 +1,60 @@
+class ContentAssetsController < ApplicationController
+  before_action :set_content_asset, only: [:show, :edit, :update, :destroy]
```

Comment:
This controller needs to be authenticated

```
before_action :authenticate_user!
```

Adding this will make your specs fail - adding a signin step should fix them

```
sign_in FactoryBot.create(:user)
```

Reply:
lets do this as a separate PR